examples/flutter_login/lib/login/view/login_form.dart

```diff
@@ -10,7 +10,7 @@ class LoginForm extends StatelessWidget {
   Widget build(BuildContext context) {
     return BlocListener<LoginBloc, LoginState>(
       listener: (context, state) {
```

Comment:
add listenWhen because after an exception when you correct your input it'll always call the snackbar.
`listenWhen: (previous, current) => previous.status != current.status,`